### ISSUE TITLE
issue with evaluate-models 

### ISSUE THREAD

**Issue by RoboticistAbdulrahman** [ChristianBirchler/sdc-scissor#66]
`poetry run sdc-scissor -c evaluate-models.yml`
[terminal output](https://github.com/ChristianBirchler/sdc-scissor/files/9220249/evaluate-models.s_bug.txt)

System information

OS: windows 11

CPU: 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz 2.30 GHz

GPU: RTX 3060

Software: version v2.2 of SDC-Scissor

**Comment by ChristianBirchler** [comment]
Hi @RoboticistAbdulrahman, thanks for the report

For the evaluation of the ML models require a more extensive set of tests since it appliest 5-fold cross-validation. You can observe the issue by looking at the log:
```
ValueError: Cannot have number of splits n_splits=5 greater than the number of samples: n_samples=4.
```

Solution: You can generate on your own more tests then it should work.

However, I will also add more sample tests so that all commands will work out of the box without generating necessarily new tests.

**Comment by ChristianBirchler** [comment]
I opened another issue (#67) you can follow.

**Comment by RoboticistAbdulrahman** [comment]
Thank you @ChristianBirchler for the feedback

**Comment by RoboticistAbdulrahman** [comment]
Thanks @ChristianBirchler, I'll test it out

**Comment by ChristianBirchler** [comment]
@RoboticistAbdulrahman, I just closed this issue since I opened #67.

**Comment by RoboticistAbdulrahman** [comment]
Thanks, Noted
